src/Microsoft.AspNetCore.Http.Connections/HttpConnectionManager.cs

```diff
@@ -147,8 +147,8 @@ public void Scan()
                 // Scan the registered connections looking for ones that have timed out
                 foreach (var c in _connections)
                 {
-                    var status = HttpConnectionContext.ConnectionStatus.Inactive;
-                    var lastSeenUtc = DateTimeOffset.UtcNow;
+                    HttpConnectionContext.ConnectionStatus status;
```

Comment:
ugh

Reply:
It's a public API. Why is it nested?

Reply:
When we get to the API review, it probably won't be public anymore.